shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -39,6 +39,19 @@ FLUTTER_EXPORT
  */
 @property(nonatomic) FlutterMouseTrackingMode mouseTrackingMode;
 
+/**
+ * Initializes this FlutterViewController with the specified `FlutterEngine`.
+ *
+ * The initialized viewcontroller will attach itself to the engine as part of this process.
+ *
+ * @param engine The `FlutterEngine` instance to attach to. Cannot be nil.
+ * @param nibName The NIB name to initialize this controller with.
+ * @param nibBundle The NIB bundle.
+ */
+- (nonnull instancetype)initWithEngine:(nonnull FlutterEngine*)engine
```

Comment:
It confuses me that this constructor is added only for tests. Can we add it to `FlutterViewController_Internal.h`, or even the test file, in the same way of defining private properties in `.mm`?

Reply:
Yeah, good idea.  Moved.